test/test.js

```diff
@@ -48,6 +48,19 @@ $.mockjax( {
 	}
 } );
 
+$.mockjax( {
```

Comment:
in this case I would prefer we would use a url which cannot conflict with other tests by mistake, e.g. adding a request param into the url with the issue id or something (just as a marker)